Readme.md

```diff
@@ -33,8 +33,6 @@ The engineering team regularly tests the extension on the following platforms:
 - x86_64-apple-darwin (MacOS 12) (tested frequently on eng workstation)
 - aarch64-apple-darwin (MacOS 12) (tested frequently on eng workstation)
 
-aarch64-unknown-linux-gnu support is [in development](https://github.com/timescale/timescaledb-toolkit/issues/382).
-
```

Comment:
Did you mean to remove it entirely?  It should be folded into the above list with the [in development] part removed.  Thanks!

Reply:
I'd say list it second, something like:
- aarch64-unknown-linux-gnu (Ubuntu Linux 20.04) (tested at release time)